common/src/main/java/com/microsoft/identity/common/internal/ui/webview/challengehandlers/ClientCertAuthChallengeHandler.java

```diff
@@ -454,6 +457,8 @@ private void verifySmartcardPin(@NonNull final char[] pin, @NonNull final YubiKi
             useSmartcardCertForAuth(certDetails.getCertificate(), pin, certDetails.getSlot().getStringAlias(), piv, request);
         } catch (final InvalidPinException e) {
             // An incorrect Pin attempt.
+            //Clear PIN attempt from local char array.
+            clearPin(pin);
```

Comment:
nit: maybe rename this to "tryUsingSmartcardWithPin" or something of sorts.
(because it's not just "verify smartcard pin", it's also doing smartcard auth here.)